src/drivers/apl-ssp.c

```diff
@@ -266,6 +266,22 @@ static inline int ssp_set_config(struct dai *dai,
 
 	sscr0 |= SSCR0_MOD | SSCR0_ACS;
 
+	/* additional hardware settings */
+	sscr1 |= (ssp->params.quirks & SOF_DAI_INTEL_SSP_QUIRK_TINTE) ?
+		SSCR1_TINTE : 0;
+	sscr1 |= (ssp->params.quirks & SOF_DAI_INTEL_SSP_QUIRK_PINTE) ?
+		SSCR1_PINTE : 0;
+	sscr1 |= (ssp->params.quirks & SOF_DAI_INTEL_SSP_QUIRK_LBM) ?
```

Comment:
So LBM is going in a more standardized route that will also build an ALSA kcontrol (so is tristate) that can be changed at runtime by userspace. Can you put a small comment above the others to say what they are for ? 
@lbetlej @plbossart can you guys comment too.

Reply:
Yeah, I already provided feedback that LBM is not a quirk, it should be enabled differently.